staging/vhost-device-can/README.md

```diff
@@ -0,0 +1,114 @@
+# vhost-device-can - CAN emulation backend daemon
+
+## Description
+This program is a vhost-user backend that emulates a VirtIO CAN device.
+The device's binary takes two (2) parameters:  a socket, a 'can-devices' list.
+The socket is commonly used across all vhost-devices to communicate with
+the vhost-user frontend device. The 'can-devices' represents a list of
+CAN/FD devices appears in the host system. This list includes *can_input* and
+*can_output* devices which vhost-device-can will forward messages to and from
+the frontend side.
+
+This program is tested with QEMU's `vhost-user-device-pci` device.
+Examples' section below.
+
+## Synopsis
+```
+**vhost-device-can** [*OPTIONS*]
+````
+
+## Options
+
+.. program:: vhost-device-can
+
+.. option:: -h, --help
+
+  Print help.
+
+.. option:: -s, --socket-path=PATH
+
+  Location of vhost-user Unix domain sockets, this path will be suffixed with
+  0,1,2..socket_count-1.
+
+.. option:: -c, --socket-count=INT
+
+  Number of guests (sockets) to attach to, default set to 1.
+
+.. option:: -d, --can-devices='CAN/FD interfaces'
+
+  CAN/FD device list at the host OS in the format:
+      <can-in_1>:<can_out_1> [<can_in_2>:<can_out_2>] ... [<can_in_N>:<can_out_N>]
+
+  Note: Where N is equal with the number provided via *socket_count* parameter.
+
+      Example: --device-list "vcan0:vcan1 vcan2:vcan3"
```

Comment:
Is this example about `--can-devices` parameter? Or, `--device-list` is another parameter?

Reply:
You are right! It's about the same thing, I just corrected that.

Reply:
Thanks, I think it should be corrected also here in the commit msg right? https://github.com/virtualopensystems/qemu/commit/077c9b9b924ee5fe0e9844659de09b862ce2d5e7